lib/eex/lib/eex/tokenizer.ex

```diff
@@ -1,6 +1,13 @@
 defmodule EEx.Tokenizer do
   @moduledoc false
 
+  @type marker :: '=' | ''
+  @type contents :: IO.chardata
+  @type line :: non_neg_integer
+  @type token :: {:text, contents} |
+                 {:expr | :start_expr | :middle_expr | :end_expr, line, marker, contents}
+  @type error :: String.t
```

Comment:
Maybe we could export a bit less types here, since we don't need all of them. For example, we can get rid of `marker/0` that we only use once and use `marker :: '=' | ''` in the `token/0` type. Wdyt?